pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -44,6 +44,15 @@ func (pa *PodAutoscaler) Class() string {
 	return autoscaling.KPA
 }
 
+// Metric returns the contents of the metric annotation or a default.
+func (pa *PodAutoscaler) Metric() string {
+	if m, ok := pa.Annotations[autoscaling.MetricAnnotationKey]; ok {
+		return m
+	}
+	// TODO: defaulting here is awkward and is already taken care off by defaulting logic.
```

Comment:
What's the action-item here?
```suggestion
	// TODO: defaulting here is awkward and is already taken care of by defaulting logic.
```